provider/types.go

```diff
@@ -225,24 +225,26 @@ var extraTypes = map[string]schema.ComplexTypeSpec{
 		},
 		Enum: []schema.EnumValueSpec{
 			{Value: "dotnet6", Name: "Dotnet6"},
-			{Value: "dotnet7", Name: "Dotnet7"},
```

Comment:
Should this be deprecated instead?

Reply:
It is, the block with deprecated runtimes is a couple of lines down (line 247 for this one)

Reply:
My bad, sorry about the noise.